src/Microsoft.DotNet.Build.Tasks/PackageFiles/ReferenceAssemblies.targets

```diff
@@ -6,7 +6,7 @@
          the third part is for bugfixes, the fourth is for hotfixes/securtity-updates -->
     <APIVersion>$([System.Version]::Parse('$(AssemblyVersion)').ToString(2))</APIVersion>
 
-    <_usingRoslyn Condition="'$(MSBuildToolsVersion)' == 'dogfood' OR '$(MSBuildToolsVersion)' &gt;= '14.0'">true</_usingRoslyn>
+    <_usingRoslyn Condition="'$(MSBuildToolsVersion)' == 'dogfood' OR '$(MSBuildToolsVersion)' == 'Current' OR '$(MSBuildToolsVersion)' &gt;= '14.0'">true</_usingRoslyn>
```

Comment:
I think you can delete this whole line and use of this property at the end of this file. I do not see `_usingRoslyn` used anywhere else.

Reply:
Yup.  I'll leave it as-is for now, as I've already tested this change and was hoping to just keep things as minimal as possible to get it working again.  But I'll open an issue to review all of this, including whether it's even necessary to do these Roslyn checks at all anymore.